examples/parameters.rs

```diff
@@ -0,0 +1,86 @@
+//! FMOD local parameter example usage:
+//! Press H and L to change the amount of traffic you can hear.
+//! Press E and M to change the time to Evening and Morning respectively.
+//! Note that because `ignore_seek_speed` is `false` in these examples,
+//! it could take a few seconds before you can hear the changes in these sound effects.
```

Comment:
To avoid confusion, I would add one more line linking to https://www.fmod.com/docs/2.00/studio/parameters-reference.html